docs/terminal/melbourne.md

```diff
@@ -206,32 +206,64 @@ Due to the low level of CTA surrounding YMMB, it is best practice to give airway
     **AAC:** "Cleared to YBLT via SAMIG, flight planned route, climb to A040, AAC"
 
 ## Flow
-The tables below give an estimated time **in minutes** from the **Feeder Fix** to the **Threshold**, which can be used to plan sequencing actions within the TCU.
-
-The times assume there is *Nil wind*. The data is for **Jets**, although there are **Non-Jet** adjustments listed below.
-
-| Feeder Fix | 09  | 16  | 27  | 34  |
-| ---------- | --- | --- | --- | --- |
-| BOYSE†     | 16  | 13  | 11  | 14  |
-| LIZZI      | 15  | 12  | 10^ | 13  |
-| WAREN      | 16  | 13  | 9^  | 10^ |
-| PORTS      | 11^ | -   | -   | 7^  |
-| WENDY      | 11^ | 17  | 15  | 12  |
-| ARBEY      | 11^ | 9^  | 12  | 13  |
-
-- †BOYSE STAR only available to Non-Jets  
-- 09A/16A IAF - Threshold is **4 minutes**  
-- 27A/34A IAF - Threshold is **3.5 minutes**  
-- Add **1 minute** for aircraft assigned a reduced speed, Except ^  
-- For **Non-Jets** (except **DH8D**);
-    - Add **2 minutes** (Except ^)
-    - Add only **1 minute** for ^
-- Subtract **1 minute** for MX or CSR  
-- Subtract **1 minute** for aircraft on the Runway 34 **Victor** STAR  
-- Subtract **2 minutes** for aircraft on the Runway 16 **Mike/Papa** STARs  
+### Local Knowledge
+- Aircraft from the south may be issued the PORTS STAR in lieu of the WAREN STAR to reduce track miles to runways 09 and 34
+- In lieu of a STAR, aircraft from the west may be radar vectored via a right circuit to runway 16 to reduce track miles
+- Aircraft from the northeast may be issued RWY 27 or direct to the IAF of the RNP-M RWY 16 to reduce track miles
```

Comment:
Just want to make sure that its clear that DCT IAF for RNP-M RWY 16 would require heads-up coord

Reply:
Added